### PR TITLE
Fixed security issue and ci

### PR DESCRIPTION
**Pull request by dorayakikun**
Fixed below issue. (and CI)

https://www.npmjs.com/advisories/786

### CODE REVIEW

**Review by bcomnes — APPROVED** [review]
Would love to see this land!

**Review by uzuko001 — APPROVED** [review]

**Comment by danielfigueiredo** [comment]
👍 to see this landing, thanks for the PR! 
There are currently two duplicated issues related to this:
https://github.com/mysticatea/cpx/issues/47
https://github.com/mysticatea/cpx/issues/48

**Comment by dorayakikun** [comment]
@mysticatea Could you review it?

**Comment by antipalindrome** [comment]
@mysticatea This would be great if we could get it. This is the only thing erroring on my audits!

**Comment by marcospgp** [comment]
@mysticatea Please have a look

**Comment by TidyIQ** [comment]
It seems like this project has been abandoned by @mysticatea 

Perhaps you could fork this and create a new npm package?

**Comment by bcomnes** [comment]
I think declaring abandon is a leap since it’s still works perfectly well. My guess is it’s a matter of notification vs abandonment. 

Has anyone tried reaching out via other channels yet?

**Comment by McSam27** [comment]
Any updates on this? Is there a maintained fork? Or is there any news from mystic?

**Comment by bcomnes** [comment]
I emailed @mysticatea a few days ago but haven't heard back.

**Comment by bcomnes** [comment]
I never heard back.  No problem.  

In the meantime, I forked to cpx2 here: https://github.com/bcomnes/cpx2/pull/1/files

It still has the `cpx` bin name, so it can be swapped out without any changes to code bases.  I use it on lots of things, so I'll try to keep it up to date as needed with greenkeeper and such.  I don't have any large refactoring or plans to change it at all.  Perhaps one day we can upstream the maintenance work ✌️

**Comment by jsomsanith-tlnd** [comment]
Hi, this PR hasn't moved since August. Do you have any idea when this will be merged/released ?
Thank you for your time and work BTW :)

**Comment by koresar** [comment]
TL;DR: the solution is to remove unmaintined `cpx` and use identical but maintained `cpx2`.

Thanks @bcomnes

**Comment by dorayakikun** [comment]
I think it will be migrated to cpx2 in the future so that I will close it.
Thanks to @bcomnes
